installer/targets/robot-user/setup

```diff
@@ -115,3 +115,56 @@ PS1='[\[\033[01;34m\]\w\[\033[00m\]]$(__git_ps1)\n'\
 '$(ROS_MASTER_NAME)${debian_chroot:+($debian_chroot)}\[\033[01;32m\]\u@\h\[\033[00m\] \$ '
 
 alias scores='scp amigo@192.168.2.110:/home/amigo/database/per_challenge/scores.html /tmp/scores.html && google-chrome /tmp/scores.html'
+
+function _github-prompt() {
+	if [[ -z $myuser || -z $mypass ]]
+	then
+		read  -p "Enter Username: " myuser
+		read -s -p "Enter Password: " mypass
```

Comment:
Yes, it is possible, but perhaps its better to re-evaluate whether this is really the way to go. 
GitHub will prompt for password when needed and with credential-helper set up properly, this won't be needed I think.